src/app/components/demo/demo.component.html

```diff
@@ -0,0 +1,72 @@
+<div>
+  <h4>LocalStorageService Demo</h4>
+  <div class="row" *ngIf="!noProviderForLocalStorage; else noLocalStorageService">
```

Comment:
it's a pretty big component to search for noLocalStorageService case, I suggest to break it into smaller components

Reply:
Done